src/core/InputOutput.ts

```diff
@@ -313,6 +313,12 @@ export async function onDownloadAs(fallback: boolean = false): Promise<boolean>
   return true;
 }
 
+export async function onCopyToClipboard(): Promise<boolean> {
+  const code = exportPathFile()!;
```

Comment:
You can return false if `exportPathFile` returned undefined. Don't use `!` here.